src/utils/__tests__/nxtAddress.test.js

```diff
@@ -1,43 +1,18 @@
 import { NxtAddress } from "utils/wallet/nxtAddress";
 
-// here's how we're using the nxtAddress code in production
-//
-// if (RSFormat) {
-//     var address = new NxtAddress();
-//
-//     if (address.set(accountId)) {
-//       return address.toString();
-//     } else {
-//       return "";
-//     }
-//   } else {
-//     return accountId;
-//   }
-// }
-//
-
-// est.mock('./library', () => {
-//     const mockExchange = { someFunction: jest.fn() };
-//     return { exchange_name: jest.fn(() => mockExchange) };
-//   });
-
-let mockNXTAddress;
-// let setSpy;
 beforeEach(() => {
-  mockNXTAddress = jest.mock("utils/wallet/nxtAddress");
-  //   setSpy = jest.spyOn(mockNXTAddress, "set");
+  jest.mock("utils/wallet/nxtAddress");
 });
 
-it.only("should create a new instance", () => {
-  const address = new NxtAddress();
+it("should create a new instance", () => {
+  new NxtAddress();
```

Comment:
NOTE: a proper test contains an expect() statement or other assertion. this seems like a nothing test. it is probably, however, still a WIP.

Reply:
yes this is WIP